public/docs/_examples/cb-a11y/ts/app/a11y-index.component.html

```diff
@@ -0,0 +1,7 @@
+<nav role="navigation">
+    <ol>
+        <li>
+            <a href="" [routerLink]="['FormControls']"><h2>Accessible form control labels</h2></a>
```

Comment:
Why the `href=""`? Doesn't `routerLink` add it dynamically? I want to make sure there isn't a fix we need to do to `routerLink` itself.